### DISCUSSION TITLE
Possibility to cancel Wifi connection

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#56] [Ideas]
When WiFi is enabled on the TEF, but signal is low or even gone, it takes too much time to be ready for using the TEF.

Proposal: allow user to cancel WiFi connection attempt.

**Comment by PE5PVB** [comment]
That is not possible. What  I can do is disable WiFi for next boot when WiFi is not found.

**Comment by andimik** [comment]
I thought you already have implemented it some days ago? After the next boot the WiFi was disabled, according to my knowledge.
